src/challenge-status/challenge-status.resolver.ts

```diff
@@ -0,0 +1,40 @@
+// import { Resolver, Mutation, Args, Query } from '@nestjs/graphql';
+// import { UseGuards } from '@nestjs/common';
+
+// import { GqlJwtAuthGuard } from '../common/guards/gql-jwt-auth.guard';
+// import { ChallengeStatusService } from './challenge-status.service';
+// import { ChallengeStatusEntity } from './entities/challenge-status.entity';
+// import { ChallengeEntity as Challenge } from 'src/challenge/entities/challenge.entity';
+// import { ChallengeStatusEntity as ChallengeStatus } from './entities/challenge-status.entity';
+// import { State } from './entities/state.enum';
+
+// @Resolver()
+// export class ChallengeStatusResolver {
```

Comment:
No need to remove the resolver. We'll still query for status of a student towards a challenge. We just don't need mutations. :-)